app/routes/authentication.py

```diff
@@ -96,6 +117,43 @@ def register_user():
     return "", 204
 
 
+@auth_blueprint.route("/user/<int:user_id>", methods=["PATCH"])
+@fresh_jwt_required
```

Comment:
Is there anywhere else we would want to use this decorator

Reply:
I can't think of another place we want to use this decorator, but if you have any ideas I'm open to hear them 🤔